### PR TITLE
fix: don't generate bzl_library targets in npm_translate_lock and npm_import by default

### PR DESCRIPTION
**Pull request by gregmagolan**
These targets are only needed in rules_js itself for a test case. Not currently needed in any downstream rule sets AFAIK and definitely not needed in user workspaces.

### CODE REVIEW

File: npm/npm_import.bzl [hunk 1]
```diff
@@ -325,6 +325,7 @@ def npm_translate_lock(
     additional_file_contents = kwargs.pop("additional_file_contents", {})
     repositories_bzl_filename = kwargs.pop("repositories_bzl_filename", None)
     defs_bzl_filename = kwargs.pop("defs_bzl_filename", None)
+    generate_bzl_library_targets = kwargs.pop("generate_bzl_library_targets", None)
```
Comment (jbedard) [inline npm/npm_import.bzl:328]: `False` instead of `None` since `generate_bzl_library_targets` is `attr.bool()`?

**Review by jbedard — APPROVED** [review]

File: npm/npm_import.bzl (hunk 1 above)
Comment (gregmagolan) [reply npm/npm_import.bzl:328]: `None` implies the default gets used which is better so the default doesn't have to be encoded in two places

Comment (jbedard) [reply npm/npm_import.bzl:328]: Isn't the second param of `pop` the value returned if the first param is not in the map?

Comment (jbedard) [reply npm/npm_import.bzl:328]: Or you mean passing `None` from here into the actual rule means it will use the rule default?

**Review by alexeagle — COMMENTED** [review]
We did add these for a reason, explained in the PR that introduced them
https://github.com/aspect-build/rules_js/pull/250
 so that you can `bazel query` to find out what exists, requested by @mattem

**Comment by gregmagolan** [comment]
Hmmm. That's problematic because of the bzl_library graph up to aspect_bazel_lib that depends on the something that doesn't exist in most cases.

**Comment by gregmagolan** [comment]
Guess we can opt-in for silo if we need them. Presumably most users won't?

**Comment by alexeagle** [comment]
I think we made a wrong turn somewhere when removing the old is_windows attribute set by wrapper macros...

**Comment by alexeagle** [comment]
Can we revert this or flip the default back?

**Pull request merged**